diagnostic.md

```diff
@@ -53,9 +68,11 @@ How would we reference each of the following categories of elements?
 -   Only those elements belonging to both the `important` and `red` _classes_
 -   All `p` elements whose parent elements are `div`s
 
-<!-- your answer starts here -->
+1. .big
+2. #contentPane
+3. .important, .red
```

Comment:
the `,` says apply this style to both of these classes. To apply the styling to ELEMENTS that have BOTH classes, you need `.important.red`